### PR TITLE
News Site by Elsa Carlström

### PR DESCRIPTION
**Pull request by elsisco**
Netlify link: https://tender-ramanujan-0cd571.netlify.app/

I'm proud of how well the responsiveness turned out and how much I have learned this week. I struggled with positioning objects inside the news cards and in the header but towards the end of the  project I feel like I understood better and could improve the code I made at the start of the project.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -4,19 +4,125 @@
...
+            </li>
+            <li class="navigation-list-item" style="display:inline">
+              <a href="" alt="1">Sport</a>
+            </li>
+            <li class="navigation-list-item" style="display:inline">
+              <a href="" alt="1">Kultur</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
```
Comment (Svempolin) [inline code/index.html:35]: very nice done!

File: code/index.html [hunk 2]
```diff
@@ -4,19 +4,125 @@
...
+        <h2>This is a headline size two</h2>
+        <p>This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph.</p>
+      </div>
+      <div class="news-cards-child blue">
+        <h4>SUBJECT</h4>
+        <img class="news-img" src="https://picsum.photos/340/170" alt="random photo" />
+        <h2>This is a headline size two</h2>
+        <p>This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph.</p>
+      </div>
+
```
Comment (Svempolin) [inline code/index.html:96]: The hole HTML is really nice done. Easy to read and to follow.

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,228 @@
...
+  width: 96%;
+  margin: auto;
+  background-color: white;
+  padding: 20px 0px 20px 0px;
+  display: flex;
+  align-content: flex-start;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
```
Comment (Svempolin) [inline code/style.css:109]: How did you ad the small blue line att the top att the news cards? NIce done!

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,228 @@
...
+  font-family: 'Inter', sans-serif;
+  font-size: 18px;
+  display: flex;
+}
+
+.big-news-parent {
+  width: 96%;
+  margin: auto;
+  background-color: white;
+  padding: 20px 0px 20px 0px;
```
Comment (Svempolin) [inline code/style.css]: this  shadow really added som nice style!

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,228 @@
...
+}
+
+.orange {
+  border-top: 4px orange solid;
+}
+
+.pink {
+  border-top: 4px hotpink solid;
+}
+
```
Comment (Svempolin) [inline code/style.css:166]: how did you make the card  "pop" up when you hovered over them? Nice Work!

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,228 @@
...
+  .news-cards-parent {
+    width: 1024px;
+    margin: auto;
+  }
+
+  .news-cards-child {
+    width: 240px;
+  }
+}
```
Comment (Svempolin) [inline code/style.css:249]: Great job Elsa! you should be proud!! :)

**Review by Svempolin — COMMENTED** [review]
So nice to se your work! Really good!!

File: code/style.css (hunk 5 above)
Comment (elsisco) [reply code/style.css:166]: Thank you for giving such a nice review!  I've seen some pages with the scale effekt when hovering - I like it too! On line 222 in my CSS you can see this `transform: scale(1.03);` which scales the news card with 3% when hovering, and on line 218 you see this `transition: transform ease-in-out .2s;` which makes the transformation possible plus the ease-in-out gives the effect a smooth transition.

File: code/style.css (hunk 3 above)
Comment (elsisco) [reply code/style.css:109]: With the border-top property! News cards with `class="red"` for example gets this `border-top: 4px red solid;` (copied from line 148 in the CSS).

**Pull request closed without merging**